Casks/j/jump.rb

```diff
@@ -1,16 +1,18 @@
 cask "jump" do
-  version "8.10.8"
-  sha256 "118979e7449850dc036bdcef7ad29e23ee4deb3c7e37cc23883eb0e037466935"
+  version "9.0.91,90091"
+  sha256 "5b0a0498273f8c8608e3be7f8791e6bc153e1d6d0c0089f062fa1dcb21bb447f"
 
-  url "https://mirror.jumpdesktop.com/downloads/jdm/JumpDesktopMac-#{version}.zip",
+  url "https://mirror.jumpdesktop.com/downloads/jdm/JumpDesktopMac-#{version.csv.second}.zip",
       user_agent: :fake
   name "Jump Desktop"
   desc "Remote desktop application"
   homepage "https://jumpdesktop.com/#jdmac"
 
   livecheck do
     url "https://mirror.jumpdesktop.com/downloads/viewer/jdmac-web-appcast.xml"
-    strategy :sparkle, &:short_version
+    strategy :sparkle do |item|
+      "#{item.short_version},#{item.version}"
+    end
```

Comment:
```suggestion
    strategy :sparkle
```

The block shouldn't be required as it should return both by default.